src/main/java/frc/robot/Commands/RunIntakeWithController.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.Commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+/* You should consider using the more terse Command factories API instead https://docs.wpilib.org/en/stable/docs/software/commandbased/organizing-command-based.html#defining-commands */
+public class RunIntakeWithController extends Command {
+  private final CommandXboxController joystick;
+  /** Creates a new RunIntakeWithController. */
+  public RunIntakeWithController( CommandXboxController joystick) {
```

Comment:
Bad constructor. Requires intakeSubsystem but is not passed in.
```suggestion
  public RunIntakeWithController( CommandXboxController joystick, IntakeSubsystem intakeSubsystem) {
```